packages/commonwealth/client/scripts/views/pages/create_community/ProtocolCommunityForm.tsx

```diff
@@ -0,0 +1,399 @@
+                  address: seedWalletMeta['address'],
+                })
+              }
+              value={seedWalletMeta.amount}
+            />
+          </>
+        )}
+      </>
+    );
+  };
+
+  return loading ? (
+    <div className="spinner-group">
+      <CWSpinner />
+      <CWText>Deploying Namespace</CWText>
+    </div>
+  ) : (
+    <div className="CreateCommunityForm">
+      <CWText type="h2" fontWeight="bold">
+        1. General Info
+      </CWText>
+      <CWDivider />
+      <CWText type="caption">
+        Set a name, description, socials and upload an image
+      </CWText>
+      <InputRow
+        title="Name"
+        placeholder="Enter the name of your community"
+        value={name}
+        onChangeHandler={(v) => {
+          setName(v);
+          setId(slugifyPreserveDashes(v));
+        }}
+      />
+      <IdRow id={id} />
+      <InputRow
+        title="Symbol"
+        value={symbol}
+        onChangeHandler={(v) => {
+          setSymbol(v);
+        }}
+      />
+      <CWText type="h2" fontWeight="bold">
+        2. Base Membership
+      </CWText>
+      <CWDivider />
+      <CWText type="caption">
+        Gate access using NFT ownership, ERC20 holders, or anyone on-chain
+      </CWText>
+      <CWDropdown
+        label="Base Chain"
+        options={[{ label: 'ethereum', value: 'ethereum' }]}
+        onSelect={(o) => {
+          setBase(o.value as ChainBase);
+        }}
+      />
+      <CWDropdown
+        label="Membership Access Options"
+        options={membershipOptions}
+        onSelect={(v) => onGateChange(v)}
+        initialValue={membershipOptions[0]}
+      />
+      {gateType !== 'all' && getGateForm(gateType)}
+      <CWText type="h2" fontWeight="bold">
+        3. Seed Your Community Wallet
+      </CWText>
+      <CWDivider />
+      <CWText type="caption">
+        Choose to link an existing Gnosis Safe/Multi-Sig, create a funding
+        proposal, or send from your wallet.
+      </CWText>
+      <CWToggle
+        checked={checked}
+        onChange={() => {
+          setChecked(!checked);
+        }}
+      />
+      {checked && seedWalletForm()}
+      <CWText type="h2" fontWeight="bold">
+        4. Socials and Additional Info
+      </CWText>
+      <CWDivider />
+      {defaultChainRows(chainFormDefaultFields)}
+      <CWButton
+        label="Save changes"
+        disabled={saving || id.length < 1}
+        onClick={async () => {
+          setSaving(true);
+
+          const additionalArgs: {
+            eth_chain_id?: number;
+            node_url?: string;
+            bech32_prefix?: string;
+            alt_wallet_url?: string;
+          } = {};
+
+          switch (base) {
+            //TODO: Goerli vars
+            case ChainBase.Ethereum:
+            default: {
+              additionalArgs.eth_chain_id = 5;
+              additionalArgs.node_url =
+                'https://eth-goerli.g.alchemy.com/v2/j4q_OFABLwfgV8b8Hel7JKLXH1y3G4_y';
```

Comment:
can you create a const (in this file or somewhere in the codebase where we keep global consts) and use it here and below, please